site/source/tutorials/accessibility.md

```diff
@@ -159,7 +159,7 @@ ___
 
 ### Dialog
 
-The Dialog widget creates a controlled dialog element. Controlled focus is not yet implemented, but will be added along with the focus manager.
+The `Dialog` widget creates a controlled dialog element. Controlled focus is not yet implemented, but will be added along with the focus manager in a future Dojo release.
```

Comment:
Maybe add a note in the ticket about this to update the tutorial when this change occurs?

Reply:
As well as any other mentions in this tutorial of future changes?

Reply:
done 👍